lib/update.php

```diff
@@ -40,12 +40,14 @@ function classicpress_override_strings( $translation, $text, $domain ) {
 
 		case 'Welcome to WordPress %1$s. You will be redirected to the About WordPress screen. If not, click <a href="%2$s">here</a>.':
 			return __(
+				/* translators: 1: omitted; 2: link to about.php (About ClassicPress screen) */
 				'Welcome to ClassicPress! You will be redirected to the About ClassicPress screen. If not, click <a href="%2$s">here</a>.',
 				'switch-to-classicpress'
 			);
 
 		case 'Welcome to WordPress %1$s. <a href="%2$s">Learn more</a>.':
 			return __(
+				/* translators: 1: omitted; 2: link to about.php (About ClassicPress screen) */
```

Comment:
The "1: omitted" here is just a mistake. I will leave this for a separate PR also, since changing it would break the translations of these strings.

Reply:
I'm not sure about this. The original line has 2 variables, but ClassicPress omits the first one. This comment helps translators understand why there is only `%2$s` variable without `%1$s`. But I might understand this wrong, as I didn't explore the filter source and this override function confuses me a bit as I don't see the passed values there.

Reply:
Yes, the original line has 2 variables, so that needs to stay as-is.

I was thinking there is no need to do the same thing in our "re-translated" string, but I guess it doesn't matter. These particular messages only appear very briefly as progress lines when the migration has finished.